### PR TITLE
Set the user_rpco_secrets file name

### PR DESCRIPTION
**Pull request by prometheanfire**
Set the variable osa_secrets_file_name so that OSA upgrades will
update the correct file.

Connects https://github.com/rcbops/u-suk-dev/issues/409

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/staging.horse/build-a8619cb6f7/docs/private-cloud/release-notes/).

**Comment by mattt416** [comment]
LGTM, I'm just unsure if this commit should be bumping SHA also?  Does anyone have anything concrete on this procedure?

**Comment by prometheanfire** [comment]
The sha bump is necessary as it includes the commit that this variable is defined in.

File: rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml [hunk 1]
```diff
@@ -152,3 +152,6 @@ repo_build_upper_constraints_overrides:
 # Increase interval between checks after container start.
 # Reduces the "Wait for container ssh" error.
 ssh_delay: 60
+
+# Use the correct secrets file
+osa_secrets_file_name: "user_rpco_secrets.yml"
```
Comment (git-harry) [inline rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml]: I think this is supposed to be 'user_osa_secrets.yml'.

**Comment by git-harry** [comment]
👎 The commit message and vars file reference user_rpco_secrets.yml but I believe the OSA secrets are in user_osa_secrets.yml.

@mattt416 I think updating the SHA is fine, it only rolls the submodule forward by one commit so we don't need to worry about cross-checking for anything else being fixed.

**Comment by prometheanfire** [comment]
they are in that file, with all the talk of combining the files I forgot all about it lol

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/staging.horse/build-a476a6742d/docs/private-cloud/release-notes/)

**Comment by mattt416** [comment]
Oops, missed the incorrect filename in original revision.  This should now be 👍 once gate runs.

File: rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml [hunk 2]
```diff
@@ -152,3 +152,6 @@ repo_build_upper_constraints_overrides:
 # Increase interval between checks after container start.
 # Reduces the "Wait for container ssh" error.
 ssh_delay: 60
+
+# Use the correct secrets file
```
Comment (git-harry) [inline rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml]: nit: mentioning this is specifically for upgrades might give a bit more clarity

**Comment by git-harry** [comment]
The commit message shouldn't have 'Fixes: ' because it will automatically close the issue, although in this case I don't think it should matter. I've taken it out of the description which I think should prevent anything happening automatically.

**Comment by git-harry** [comment]
recheck_swift unrelated apt failure.

**Comment by prometheanfire** [comment]
@git-harry does that mean you want me to resubmit with edits?

**Comment by git-harry** [comment]
@prometheanfire only nits really, so only if you have time but I don't think they're vital.

**Comment by major** [comment]
I'm :+1: if this pleases Jenkins.

**Pull request merged**